uiServices/services.go

```diff
@@ -59,31 +66,33 @@ func FilePathExist(path string) bool {
 	_, err := os.Stat(path)
 	if os.IsNotExist(err) {
 		FileExist = false
+
 	}
 	return FileExist
 }
 
-func validdateJsonWithSchema(config []byte) bool {
+func validateJsonWithSchema(config []byte) bool {
 	goPath := os.Getenv("GOPATH")
+	fmt.Println(goPath)
```

Comment:
remove code you used to test your functions. we don't want fmt.Println in our shipped code.